05.asynchronous_sqlite3/promise_with_error.js

```diff
@@ -0,0 +1,57 @@
+import sqlite3 from "sqlite3";
+import { run, each, close } from "./functions.js";
+
+const db = new sqlite3.Database(":memory:");
+
+run(
+  db,
+  "CREATE TABLE books (id INTEGER PRIMARY KEY, title TEXT NOT NULL, content TEXT)"
+)
+  .then(() =>
+    run(
+      db,
+      "INSERT INTO books (title, content) VALUES (?, ?)",
+      null,
+      "content1"
+    )
+  )
+  .then((record) => console.log(`lastID: ${record.lastID}`))
+  .catch((err) => console.error("Error発生1", err.message))
+  .then(() =>
+    run(
+      db,
+      "INSERT INTO books (title, content) VALUES (?, ?)",
+      null,
+      "content2"
+    )
+  )
+  .then((record) => console.log(`lastID: ${record.lastID}`))
+  .catch((err) => console.error("Error発生2", err.message))
+  .then(() =>
+    run(
+      db,
+      "INSERT INTO books (title, content) VALUES (?, ?)",
+      null,
+      "content3"
+    )
+  )
+  .then((record) => console.log(`lastID: ${record.lastID}`))
+  .catch((err) => console.error("Error発生3", err.message))
+  .then(() =>
+    run(
+      db,
+      "INSERT INTO books (title, content) VALUES (?, ?)",
+      null,
+      "content4"
+    )
+  )
+  .then((record) => console.log(`lastID: ${record.lastID}`))
+  .catch((err) => console.error("Error発生4", err.message))
+  .then((record) =>
```

Comment:
なんか変な `record` がいる気がします 👀 

```suggestion
  .then(() =>
```

Reply:
意味のない record でした。削除しました🙇‍♂️